public/video-ui/src/actions/VideoActions/getVideos.js

```diff
@@ -51,22 +51,34 @@ export function getVideos() {
   };
 }
 
+function adaptCapiAtom(atom) {
+  const ret = {
+    id: atom.id,
+    activeVersion: -1, // not known
```

Comment:
There are atoms in the DB that don't have that property (I think they are ones that have just been created and not published?) so I wanted to make an explicit distinction between those and atoms that have come from search results.